.github/workflows/test.yml

```diff
@@ -29,6 +29,13 @@ jobs:
       - name: Run linters
         run: |
           make lint
+      - name: Install type checking dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install django==4.2.10 django-stubs==4.2.7 mypy==1.8.0
```

Comment:
I see you're using django-stubs, but no mypy.ini with configured plugin. Not using plugin probably ok if we aren't using anything requiring plugin.

But let's add `mypy.ini` with at least this

```ini
[mypy]
incremental = True
warn_unused_ignores = True
warn_redundant_casts = True
warn_unused_configs = True
pretty = True
show_error_context = True
```

Reply:
TIL about `warn_redundant_casts`, really useful